### ISSUE TITLE
7.3.1 - Default condition should be last one

### ISSUE THREAD

**Issue by davidye** [pixijs/pixijs#9691]
### Current Behavior

Updated to 7.3.1 and getting this compilation error:

./node_modules/@pixi/color/lib/Color.mjs:1:0
Module not found: Default condition should be last one

### Expected Behavior

No compilation error

### Steps to Reproduce

Seems related to the hotfix?

### Environment

- **`pixi.js` version**: *e.g. 7.1.0*
- **Browser & Version**: *e.g. Chrome 108*
- **OS & Version**: *e.g. Ubuntu 22.04*
- **Running Example**: *e.g. <https://pixiplayground.com/>*

**Comment by bigtimebuddy** [comment]
How are you getting this error? Can you please provide more information.

**Comment by Zyie** [comment]
> How are you getting this error? Can you please provide more information.

Seems like this might be a webpack issue

https://github.com/firebase/firebase-js-sdk/pull/7007

I think I just need to move each `default` to be below `types`
https://github.com/pixijs/colord/blob/master/package.json#L35

**Comment by Zyie** [comment]
I've published a new version of `colord` with that change, so reinstalling pixi should fix this as we use `"@pixi/colord": "^2.9.4"` so i don't think we need a new release

**Comment by Zyie** [comment]
Confirmed by a user in the discord that this is working for them

@davidye You should just be able to uninstall/reinstall pixi, or maybe `npm update pixi.js` so it grabs the latest dependency

**Comment by MasatoMakino** [comment]
Dear pixi.js community

Regarding package.json of @pixi/colord v2.9.5, the export field of the root directory `.` seems to be missing from the fix.

https://github.com/pixijs/colord/blob/400a40e956230c802044c916d39463e66edaf70a/package.json#L32-L42

In local, the problem was corrected when rewritten as follows.

```json
  "exports": {
    ".": {
      "import": {
        "types": "./index.d.ts",
        "default": "./index.mjs"
      },
      "require": {
        "types": "./index.d.ts",
        "default": "./index.js"
      }
    },
```

I hope this helps the pixi.js community.

**Comment by Zyie** [comment]
> Dear pixi.js community
> 
> Regarding package.json of @pixi/colord v2.9.5, the export field of the root directory `.` seems to be missing from the fix.
> 
> https://github.com/pixijs/colord/blob/400a40e956230c802044c916d39463e66edaf70a/package.json#L32-L42
> 
> In local, the problem was corrected when rewritten as follows.
> 
> ```json
>   "exports": {
>     ".": {
>       "import": {
>         "types": "./index.d.ts",
>         "default": "./index.mjs"
>       },
>       "require": {
>         "types": "./index.d.ts",
>         "default": "./index.js"
>       }
>     },
> ```
> 
> I hope this helps the pixi.js community.

Hey @MasatoMakino thanks for this
I must have missed updating this export last night

I've released `2.9.6` with this fix
